diff --git a/CNAME b/CNAME
old mode 100644
new mode 100755
diff --git a/README.md b/README.md
old mode 100644
new mode 100755
diff --git a/about.html b/about.html
old mode 100644
new mode 100755
diff --git a/css/example.css b/css/example.css
old mode 100644
new mode 100755
diff --git a/css/exp.css b/css/exp.css
old mode 100644
new mode 100755
diff --git a/css/green.css b/css/green.css
old mode 100644
new mode 100755
diff --git a/css/jacob.css b/css/jacob.css
old mode 100644
new mode 100755
diff --git a/css/jemdoc.css b/css/jemdoc.css
old mode 100644
new mode 100755
diff --git a/css/nolines.css b/css/nolines.css
old mode 100644
new mode 100755
diff --git a/css/page.css b/css/page.css
old mode 100644
new mode 100755
diff --git a/css/prob.css b/css/prob.css
old mode 100644
new mode 100755
diff --git a/css/table.css b/css/table.css
old mode 100644
new mode 100755
diff --git a/css/white.css b/css/white.css
old mode 100644
new mode 100755
diff --git a/img/megha.jpg b/img/megha.jpg
old mode 100644
new mode 100755
diff --git a/index.html b/index.html
old mode 100644
new mode 100755
diff --git a/jemdoc/MENU b/jemdoc/MENU
old mode 100644
new mode 100755
diff --git a/jemdoc/Makefile b/jemdoc/Makefile
old mode 100644
new mode 100755
diff --git a/jemdoc/about.jemdoc b/jemdoc/about.jemdoc
old mode 100644
new mode 100755
diff --git a/jemdoc/css/example.css b/jemdoc/css/example.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/exp.css b/jemdoc/css/exp.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/green.css b/jemdoc/css/green.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/jacob.css b/jemdoc/css/jacob.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/jemdoc.css b/jemdoc/css/jemdoc.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/nolines.css b/jemdoc/css/nolines.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/page.css b/jemdoc/css/page.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/prob.css b/jemdoc/css/prob.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/table.css b/jemdoc/css/table.css
old mode 100644
new mode 100755
diff --git a/jemdoc/css/white.css b/jemdoc/css/white.css
old mode 100644
new mode 100755
diff --git a/jemdoc/img/megha.jpg b/jemdoc/img/megha.jpg
old mode 100644
new mode 100755
diff --git a/jemdoc/index.jemdoc b/jemdoc/index.jemdoc
old mode 100644
new mode 100755
diff --git a/jemdoc/jemdoc.conf b/jemdoc/jemdoc.conf
old mode 100644
new mode 100755
diff --git a/jemdoc/publications.jemdoc b/jemdoc/publications.jemdoc
old mode 100644
new mode 100755
index b9182d8..7e321b6
--- a/jemdoc/publications.jemdoc
+++ b/jemdoc/publications.jemdoc
@@ -2,5 +2,15 @@
 = Megha Parhi -- Publications
 
 == Conference Publications
+ . Yin Liu, *Megha Parhi*, Marc D. Riedel, and Keshab K. Parhi. [http://ieeexplore.ieee.org/document/7869017/ Synthesis of correlated bit streams for stochastic computing]. 
+ 50th Asilomar Conference on Signals, Systems and Computers, ACSSC 2016, Pacific Grove, CA, USA, November 6-9, 2016, pages 167–174. IEEE, 2016. 
+ . *Megha Parhi*, Marc D. Riedel, and Keshab K. Parhi.  [http://ieeexplore.ieee.org/document/7251915/ Effect of bit-level correlation in stochastic computing]. 
+ In 2015 IEEE International Conference on Digital Signal Processing, DSP 2015, Singapore, July 21-24, 2015, 
+ pages 463–467. IEEE, 2015. 
+ . *Megha Parhi*, Yingjie Lao, and Keshab K. Parhi.  [http://ieeexplore.ieee.org/document/7094662/ Canonic real-valued FFT structures]. 
+ 48th Asilomar Conference on Signals, Systems and Computers, ACSSC 2014, Pacific Grove, CA, 
+ USA, November 2-5, 2014, pages 1261–1265. IEEE, 2014. 
+
+ 
+
 
-Fill these in later.
diff --git a/publications.html b/publications.html
old mode 100644
new mode 100755
index c14492c..7194a0b
--- a/publications.html
+++ b/publications.html
@@ -35,10 +35,22 @@ _gaq.push(['_trackPageview']);
 <h1>Megha Parhi &ndash; Publications</h1>
 </div>
 <h2>Conference Publications</h2>
-<p>Fill these in later.</p>
+<ol>
+<li><p>Yin Liu, <b>Megha Parhi</b>, Marc D. Riedel, and Keshab K. Parhi. <a href="http://ieeexplore.ieee.org/document/7869017/" target=&ldquo;blank&rdquo;>Synthesis of correlated bit streams for stochastic computing</a>. 
+50th Asilomar Conference on Signals, Systems and Computers, ACSSC 2016, Pacific Grove, CA, USA, November 6-9, 2016, pages 167–174. IEEE, 2016. </p>
+</li>
+<li><p><b>Megha Parhi</b>, Marc D. Riedel, and Keshab K. Parhi.  <a href="http://ieeexplore.ieee.org/document/7251915/" target=&ldquo;blank&rdquo;>Effect of bit-level correlation in stochastic computing</a>. 
+In 2015 IEEE International Conference on Digital Signal Processing, DSP 2015, Singapore, July 21-24, 2015, 
+pages 463–467. IEEE, 2015. </p>
+</li>
+<li><p><b>Megha Parhi</b>, Yingjie Lao, and Keshab K. Parhi.  <a href="http://ieeexplore.ieee.org/document/7094662/" target=&ldquo;blank&rdquo;>Canonic real-valued FFT structures</a>. 
+48th Asilomar Conference on Signals, Systems and Computers, ACSSC 2014, Pacific Grove, CA, 
+USA, November 2-5, 2014, pages 1261–1265. IEEE, 2014. </p>
+</li>
+</ol>
 <div id="footer">
 <div id="footer-text">
-Page generated 2017-09-10 00:21:51 CDT
+Page generated 2017-09-10 10:39:14 CDT
 </div>
 </div>
 </td>
